addons/m252_mortar/model.cfg

```diff
@@ -1,40 +1,119 @@
-
-class CfgSkeletons
-{
-    class Default 
-    {
+class CfgSkeletons {
+    class Default {
         isDiscrete = 0;
         skeletonInherit = "";
         skeletonBones[] = {};
     };
-    class M252_Mortar_Skeleton: Default 
-    {
+    class M252_Mortar_Skeleton: Default {
         skeletonInherit = "";
-        skeletonBones[] = 
-        {
-
+        skeletonBones[] = {
+            "base_plate","",
+            "mortar_tube","base_plate",
+            "support_top","mortar_tube",
+            "support_base","support_top",
+            "deflection_handwheel","support_top",
+            "deflection_grip","deflection_handwheel",
+            "elevation_handwheel","support_base",
+            "elevation_grip","elevation_handwheel"
         };
     };
 };
 
-class CfgModels 
-{
-    class Default 
-    {
+class CfgModels {
+    class Default {
         sectionsInherit = "";
         sections[] = {};
         skeletonName = "";
     };
-    
-    class m252_mortar: Default 
-    {
+
+    class m252_mortar: Default {
         skeletonName = "M252_Mortar_Skeleton";
         sectionsInherit = "Default";
-        sections[] = {"camo1","camo2","camo3"};
-        
-        class Animations
-        {
-        
+        sections[] = {"camo1"};
+
+        class Animations {
+          class main_gun {
```

Comment:
[For reference](https://ace3mod.com/wiki/development/coding-guidelines.html#52-indents)

There is a `.editorconfig` settings file in this repository so I suggest simply using [the extension](http://editorconfig.org/) for your text editor to avoid these issues in future :+1: